libvast/src/format/pcap.cpp

```diff
@@ -282,8 +276,8 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
       return finish(f, caf::make_error(ec::end_of_input, "reached end of "
                                                          "trace"));
     if (r == -1) {
-      auto err = std::string{::pcap_geterr(pcap_)};
-      ::pcap_close(pcap_);
+      auto err = std::string{::pcap_geterr(pcap_.get())};
+      ::pcap_close(pcap_.get());
       pcap_ = nullptr;
```

Comment:
This is just `pcap_ = nullptr` to invoke the new deleter.

Reply:
Good spot. Just fixed.